airflow/providers/google/cloud/hooks/compute.py

```diff
@@ -72,6 +76,350 @@ def get_conn(self):
             self._conn = build('compute', self.api_version, http=http_authorized, cache_discovery=False)
         return self._conn
 
+    def get_compute_instance_template_client(self):
+        """Returns Compute Engine Instance Template Client."""
+        return InstanceTemplatesClient(credentials=self._get_credentials(), client_info=self.client_info)
+
+    def get_compute_instance_client(self):
+        """Returns Compute Engine Instance Client."""
+        return InstancesClient(credentials=self._get_credentials(), client_info=self.client_info)
+
+    def get_compute_instance_group_managers_client(self):
+        """Returns Compute Engine Instance Group Managers Client."""
+        return InstanceGroupManagersClient(credentials=self._get_credentials(), client_info=self.client_info)
+
+    @GoogleBaseHook.fallback_to_default_project_id
+    def insert_instance_template(
+        self,
+        body: dict,
+        request_id: Optional[str] = None,
+        project_id: str = PROVIDE_PROJECT_ID,
+        retry: Optional[Retry] = None,
+        timeout: Optional[float] = None,
+        metadata: Optional[Sequence[Tuple[str, str]]] = (),
+    ) -> Any:
```

Comment:
Maybe it is better to return None not an Any, because this hook does not return anything?

Reply:
Done